script/utils/verifyContracts.sh

```diff
@@ -23,7 +23,7 @@ function verify_contract {
     if [[ $verifier_url == *"api."* ]]; then
         verifierArgs="$verifierArgs --verifier-api-key $verifier_api_key --verifier=etherscan"
     elif [[ $verifier_url == *"explorer."* || $verifier_url == *"blockscout."* ]]; then
-        verifierArgs="$verifierArgs --verifier=blockscout"
+        verifierArgs="$verifierArgs --verifier-api-key \"\" --verifier=blockscout"
```

Comment:
The `verifier_url` heuristic seems brittle to me. How about we leave it for now, but if it ever breaks instead of adding more hacks we change to a `VERIFIER_MODE` setting?

Reply:
sounds good!